app/services/create_transaction.rb

```diff
@@ -34,4 +35,10 @@ def convert_and_assign_value(transaction, value)
   rescue ConvertFinancialValue::Error
     transaction.errors.add(:value, I18n.t("activerecord.errors.models.transaction.attributes.value.not_a_number"))
   end
+
+  def default_description
+    return nil unless transaction.date.present?
+
+    "#{transaction.financial_quarter_and_year} spend on #{activity.title}"
```

Comment:
Does it matter that this will go out of sync if `activity.title` is changed in the future?

Reply:
It's a great point, my view is that the title was accurate at the time the transaction was added and that is absolutely fine.